funding/manager.go

```diff
@@ -684,6 +680,36 @@ func (f *Manager) Stop() error {
 	return nil
 }
 
+// rebroadcastFundingTx publishes the funding tx on startup for each
+// unconfirmed channel.
+func (f *Manager) rebroadcastFundingTx(c *channeldb.OpenChannel) {
+	var fundingTxBuf bytes.Buffer
+	err := c.FundingTxn.Serialize(&fundingTxBuf)
+	if err != nil {
+		log.Errorf("Unable to serialize funding transaction %v: %v",
+			c.FundingTxn.TxHash(), err)
+
+		// Clear the buffer of any bytes that were written before the
+		// serialization error to prevent logging an incomplete
+		// transaction.
+		fundingTxBuf.Reset()
+	}
+
+	log.Debugf("Rebroadcasting funding tx for ChannelPoint(%v): %x",
```

Comment:
Have this in the else clause so we don't log nothing if we can't serialize it (which should never really happen in practice).